packages/services/api/src/modules/audit-logs/module.graphql.ts

```diff
@@ -0,0 +1,236 @@
+import { gql } from 'graphql-modules';
+
+export const typeDefs = gql`
+  interface AuditLog {
+    id: ID!
+    eventTime: DateTime!
+    user: AuditLogUserRecord!
+    organizationId: String!
+  }
+
+  type AuditLogUserRecord {
+    userId: ID!
+    userEmail: String!
+    user: User # this one is nullable because it can be deleted!
+  }
+
+  type AuditLogConnection {
+    nodes: [AuditLog]!
+    total: Int!
+  }
+
+  type AuditLogFileExport {
+    id: ID!
+    url: String!
+    validUntil: DateTime!
+    createdAt: DateTime!
+  }
+
+  type ExportResultEdge {
+    node: AuditLogFileExport!
+    cursor: String!
+  }
+
+  type ExportResultConnection {
+    edges: [ExportResultEdge!]!
+    pageInfo: PageInfo!
+  }
+
+  input AuditLogFilter {
```

Comment:
reduce the filter for the basics of filtering, and later we can improve it with the UI.